css/modal.css

```diff
@@ -0,0 +1,126 @@
+.modal {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: 55%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  border: 1px solid var(--white-color);
+  border-radius: 5px;
+  z-index: 5;
+  background-color: var(--primary-color);
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  width: 80%;
+  padding: 0 1.5rem;
+}
```

Comment:
- [ ] Great job implementing the Modal, but there seem to be slight differences compared to the Figma template. Your Modal has a different opacity/blur and background color compared to the Figma template as you can see below
> Yours
![image](https://user-images.githubusercontent.com/90556221/167740251-5d9aff6b-bbde-4df6-8002-5aa41c4e821e.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/167740293-0d33ae9a-d15c-4408-b6e4-64b0bca6e32b.png)

Kindly alter it to conform to the Figma template as much as possible.

- [ ] Additionally, your technologies seem to have stray commas between them, kindly remove them as they ruin the visual integrity of your Modal
![image](https://user-images.githubusercontent.com/90556221/167740443-2257d66f-40ab-4427-b120-c773374937c7.png)